tests/x11regressions/gnomecase/login_test.pm

```diff
@@ -29,7 +29,11 @@ sub run () {
 
     assert_screen "generic-desktop";
     $self->auto_login_alter;
+    my $ov = get_var('NOAUTOLOGIN');
+    set_var('NOAUTOLOGIN', '');
     reboot_gnome;
+    wait_boot;
+    set_var('NOAUTOLOGIN', $ov);
```

Comment:
it is a very special case for this test, so I don't want to make wait_boot more complex. Having a comment is fine